modules/core/field/src/FarmFieldFactory.php

```diff
@@ -781,6 +781,14 @@ protected function modifyIntegerField(BaseFieldDefinition &$field, array $option
       $field->setSetting('size', $options['size']);
     }
 
+    // Set the min/max constraints, if specified.
+    if (!empty($options['min'])) {
```

Comment:
`!empty()` will not work for a min/max value of `0`. I think we should use `isset()`.

Reply:
Oh good point @paul121 thanks for reviewing! I'll make this change and re-request review.